ProteoformSuiteInternal/inputFile.cs

```diff
@@ -66,20 +66,15 @@ public InputFile(string completePath, Labeling label, Purpose purpose)
         }
     }
 
-    //for TD 
-    //public enum TDProgram
-    //{
-    //    ProSight,
-    //    NRTDP //NU software
-    //}
-
     public enum Purpose
     {
         Identification,
         Quantification,
```

Comment:
Calibration is still being used in the lab. Probably good to add it back for now.